usr/src/test/zfs-tests/tests/functional/acl/acl_common.kshlib

```diff
@@ -202,7 +202,7 @@ function plus_sign_check_v #<obj>
 		return 1
 	fi
 
-	ls -vd $obj | nawk '(NR == 1) {print $1}' | grep "+\>" > /dev/null
+	ls -vd $obj | nawk '(NR == 1) {print $1}' | grep "+$" > /dev/null
```

Comment:
why in line 188 we are using awk , but on this line - nawk ?
maybe fix it in both cases to awk or nawk?

Reply:
Sure.